### PR TITLE
added a new visdom task

### PR DESCRIPTION
**Pull request by ImenBizid**
added a generic task to visualize the results obtained by a predictive model
updated the metadata of the data-visualization package

### CODE REVIEW

File: DataVisualization/resources/catalog/Visdom_Visualize_Results.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="https://s3.eu-west-2.amazonaws.com/activeeon-public/icons/visdom.png"/>
+      </genericInformation>
+      <forkEnvironment javaHome="${PA_SCHEDULER_HOME}/jre/" >
+        <envScript>
+          <script>
+            <code language="python">
+              <![CDATA[
+#Be aware, that the prefix command is internally split by spaces. So paths with spaces won't work.
```
Comment (amouhoub) [inline DataVisualization/resources/catalog/Visdom_Visualize_Results.xml]: Is this comment relevant to this ML task?

Comment (ImenBizid) [reply DataVisualization/resources/catalog/Visdom_Visualize_Results.xml]: It was added by default by the Proactive Studio
